src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -96,6 +96,12 @@ function CatalogueCardView(props: CatalogueCardViewProps) {
         bottom={12}
         right={0}
       >
+        <Grid item position={'fixed'} left={8}>
```

Comment:
This causes the navigation drawer to be rendered over it rather than allowing it to move out of the way.